test/test-mongocrypt-ctx-setopt.c

```diff
@@ -485,10 +484,6 @@ _test_setopt_for_encrypt (_mongocrypt_tester_t *tester)
    ENCRYPT_INIT_OK ("a", -1, cmd);
    MASTERKEY_LOCAL_FAILS ("cannot set options after init");
 
-   REFRESH;
-   ENDPOINT_OK ("example.com:80", -1);
-   ENCRYPT_INIT_FAILS ("a", -1, cmd, "endpoint prohibited");
```

Comment:
endpoint is contained within the `_mongocrypt_kek_t.providers` union. There is no longer an explicit check in other contexts to ensure it hasn't been incorrectly set by the wrapping driver. This would only be checking against API misuse by the driver, so I'm not concerned that there is no longer a check.